dev-db/docker-compose.yml

```
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  db:
+    image: "postgres:13"
+    container_name: "my_postgres"
+    environment:
+      POSTGRES_USER: ${DB_USER}
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+    ports:
+      - "5432:5432"
+    volumes:
+      - my_dbdata:/var/lib/postgresql/dataß
```

Is this a special character at the end and is it intended?

---

Hmm..not sure how that got there, but no it's not intentional.
```suggestion
      - my_dbdata:/var/lib/postgresql/data
```